### PR TITLE
config: use hagezi dynamic dns and rmv nextdns & shadowwhisperer bypass

### PR DESCRIPTION
**Pull request by SpencerIsGiddy**
Replaces the dynamic dns source of nextdns  with hagezi’s which includes nextdns’s source

https://github.com/hagezi/dns-blocklists/blob/main/usedsources.md#dyndns

also removes a ton of the overlapping bypass method lists due to hagezi’s list already combining them already.

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -4,17 +4,8 @@
       "vname": "Prevent bypass",
       "group": "ParentalControl",
       "subg": "BypassMethods",
-      "format": ["domains", "domains", "domains", "domains", "domains",
-                 "domains", "domains", "domains", "wildcard", "domains"],
+      "format": ["wildcard", "domains"],
       "url": [
-        "https://raw.githubusercontent.com/nextdns/dns-bypass-methods/main/apple-private-relay",
```
Comment (ignoramous) [inline config.json:10]: these are removed because dyn-dns covers them? If so, shouldn't these in bypass methods and not dyn-dns?

Comment (GiddyGoatGaming) [reply config.json:10]: so the DoH/VPN/TOR/Proxy Bypass Hagezi list includes all the lists removed in the prevent bypass section.

 The dyn dns Hagezi list includes the nextdns list that was there but with more so thats why that source is switched

Comment (ignoramous) [reply config.json:10]: Oh you mean `https://raw.githubusercontent.com/hagezi/dns-blocklists/main/wildcard/doh-vpn-proxy-bypass.txt` has all the lists this commit removes?

Comment (SpencerIsGiddy) [reply config.json:10]: Yes

**Pull request merged**